src/patternfly/components/Button/examples/button-types-example.hbs

```diff
@@ -15,5 +15,5 @@
     Link Button
 {{/button}}
 {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Remove"'}}
-    {{> button-icon button-icon--type="times"}}
+    {{> button-icon button-icon--type="times" button--attribute='aria-label="Remove"'}}
```

Comment:
Can you revert the change for this file so that the PR shows no changes to this file?

Reply:
Reverted the code back for this one as well.